testutil/testutil.go

```diff
@@ -95,3 +96,10 @@ func PrintMetrics(m []telegraf.Metric) {
 	}
 	fmt.Println(string(buf))
 }
+
+// DefaultSampleConfig returns the sample config with the default parameters
+// uncommented to also be able to test the validity of default setting.
+func DefaultSampleConfig(sampleConfig string) []byte {
+	re := regexp.MustCompile(`(?m)(^\s+)#\s*`)
+	return []byte(re.ReplaceAllString(sampleConfig, "$1"))
+}
```

Comment:
Would it be OK for you to split this out to another PR in order to not hold-up its use?

Reply:
I would love more this PR to finally land..